src/client/sandbox/node/window.js

```diff
@@ -183,6 +184,17 @@ export default class WindowSandbox extends SandboxBase {
             };
         }
 
+        window.Range.prototype.createContextualFragment = function () {
```

Comment:
It is wrong. In 9 `window.Range.prototype` does not contains method `createContextualFragment`.
Need to change on:
```javascript
if (window.Range.prototype.createContextualFragment) {
    window.Range.prototype.createContextualFragment = function (){...}
}
```